cypress/e2e/pap-home/pap-home-servizi.cy.ts

```diff
@@ -0,0 +1,50 @@
+import {homeButtons, serviziButtons} from 'projects/pap/src/app/features/home/home.model';
+
+const serviziButton = homeButtons.find(button => button.label === 'Servizi');
```

Comment:
il codice è scritto in inglese e quindi anche le variabili DEVONO essere scritte in inglese